src/tx/applyTx/checkSpendCond.js

```diff
@@ -27,6 +27,7 @@ const {
   ERC20_BYTECODE,
   ERC721_BYTECODE,
   ERC1948_BYTECODE,
+  ERC1948_BYTECODE_OLD,
```

Comment:
Maybe use the naming like `ERC1948_BYTECODE_41000` as it may not be the last switch ?